example/src/components/ThirdPartyVectorTileSource.js

```diff
@@ -9,15 +9,14 @@ import sheet from '../styles/sheet';
 const styles = MapboxGL.StyleSheet.create({
   line: {
     lineCap: "round",
-    lineJoin: "rouncd",
+    lineJoin: "round",
     lineOpacity: 0.6,
     lineColor: "rgb(53, 175, 109)",
     lineWidth: 2
   },
 });
 
-const VECTOR_SOURCE_URL =
-  'https://d25uarhxywzl1j.cloudfront.net/v0.1/{z}/{x}/{y}.mvt';
+const TILEJSON_URL = "https://s3.amazonaws.com/tgaw/tilejson.json";
```

Comment:
I'm not sure if we'll want to leave this URL in place. That's a file to an object in an S3 bucket that I own.

Reply:
yea something we can do is just add a local tilejson file to the example app, so we can pull it off of the devices file system

Reply:
Oh sweet, didn't realize that could be a local file. I can update the example to use that instead. Also need to update my app to do the same.